src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
```

Comment:
**Issue**: Initialization of 'absMax' to 0 can lead to incorrect results when all numbers are negative. <br> **Fix**: Initialize 'absMax' with the first element of the array to ensure correct behavior across all possible input scenarios. <br> **Code Suggestion**: 
 ```
 16c16
 <         int absMax = 0;
 ---
 >         int absMax = numbers[0];
 ```